specification/monitor/resource-manager/Microsoft.Insights/preview/2017-05-01-preview/subscriptionDiagnosticsSettings_API.json

```diff
@@ -0,0 +1,341 @@
+{
+	"swagger": "2.0",
+	"info": {
+		"title": "MonitorManagementClient",
+		"x-ms-code-generation-settings": {
+			"name": "MonitorManagementClient"
+		},
+		"version": "2017-05-01-preview"
```

Comment:
It seems odd to add this only in a 2.5 year old preview version

Reply:
The development of this feature happened in stages over time.